packs/dotnet/dotnet/alerts/ADPEXScore.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+name: ADPEX Score
+
+details: |+
+  This alert is triggered when ADPEX Score is below 0.4 for 5 minutes
+type: STATIC
+
+nrql:
+  query: "SELECT apdex(apm.service.apdex) as 'App server' FROM Metric WHERE (entity.guid = 'MjU5NDA4MnxBUE18QVBQTElDQVRJT058MTQ1NzUxMzQ3')"
```

Comment:
This query is not going to work because the guid will be unique for every user. 
Is there a more generic WHERE clause we can use? For example, is there a way to filter on .net applications? Otherwise I think we will have to remove the WHERE clause all together

Reply:
On "Transactions" and "Errors" tabs it works fine without the WHERE clause but when i try to do it on my VM Metrics i get this error: 'We can't render this query: No application was matched (did you specify appId, appName or entity.guid?)' it seems like I need to point toward something. I have not found a way to work around that yet.

Reply:
WHERE appName like '%' should work